ElectronClient/gui/NoteText.jsx

```diff
@@ -1512,11 +1512,23 @@ class NoteTextComponent extends React.Component {
 	}
 
 	commandTextBold() {
-		this.wrapSelectionWithStrings('**', '**', _('strong text'));
+		const selection = this.textOffsetSelection();
+		let string = this.state.note.body.substr(selection.start, selection.end - selection.start);
+		if (string.startsWith('**') && string.endsWith('**')) {
+			this.wrapSelectionWithStrings('', '', '', string.substr(2, selection.end - selection.start - 4));
+		} else {
+			this.wrapSelectionWithStrings('**', '**', _('strong text'));
+		}
 	}
 
 	commandTextItalic() {
-		this.wrapSelectionWithStrings('*', '*', _('emphasized text'));
+		const selection = this.textOffsetSelection();
+		let string = this.state.note.body.substr(selection.start, selection.end - selection.start);
+		if (string.startsWith('*') && string.endsWith('*')) {
```

Comment:
Please also check for underscore here.
`if ((string.startsWith('*') && string.endsWith('*')) || (string.startsWith('_') && string.endsWith('_'))) {`

Reply:
@tessus I've made the requested changes and update the issue. Please review.